jobs/event_queue.go

```diff
@@ -0,0 +1,48 @@
+package jobs
+
+import (
+	"time"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/duty/models"
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+)
+
+// CleanupEventQueue deletes state records in the `event_queue` table
+func CleanupEventQueue() {
```

Comment:
Rename to CleanupPushEvents or something

Reply:
I think we would want to clean up other event types in the queue as well.

Reply:
Added deletion of other events as well